docs/reverse_proxy.md

```diff
@@ -42,23 +42,53 @@ the reverse proxy and the homeserver.
             location /_matrix {
                 proxy_pass http://localhost:8008;
                 proxy_set_header X-Forwarded-For $remote_addr;
+            }            
+            location /.well-known/matrix/server {
+                proxy_pass http://localhost:8008;
+                proxy_set_header X-Forwarded-For $remote_addr;
+            }
+            location /.well-known/matrix/client {
+                proxy_pass http://localhost:8008;
+                proxy_set_header X-Forwarded-For $remote_addr;
+            }
+            location /.well-known {
+                proxy_pass http://localhost:8008;
+                proxy_set_header X-Forwarded-For $remote_addr;
             }
```

Comment:
surely you don't need *all* of these?

Reply:
Look, without those nothing works, especially if the domain is used at / by another webapp(Pleroma for example), with those setted up I can use web clients for matrix without errors.

Reply:
that doesn't sound right to me, I'm afraid.